advanced-integration/server.js

```diff
@@ -59,9 +59,9 @@ app.get("/check" ,(req,res) => {
   res.json({
     message: "ok",
     env: process.env.NODE_ENV, 
-    clientId: process.env.CLIENT_ID,
-    appSecret: process.env.APP_SECRET || "Couldn't load App Secret",
-    merchantId: process.env.MERCHANT_ID,
+    clientId: process.env.PAYPAL_CLIENT_ID,
+    appSecret: process.env.PAYPAL_CLIENT_SECRET || "Couldn't load App Secret",
```

Comment:
This is unrelated to this PR, but I wanted to mention that the client secret should never be sent back to the browser. 
```suggestion
```

@RatnadeepSimhadri can we remove passing the client secret back to the browser?

Reply:
Yes it shouldn't be, Removed all the values.